a8c-files.php

```diff
@@ -104,6 +104,9 @@ public function __construct() {
 				$this->schedule_update_job();
 			}
 		}
+
+		// Add filter to fix image block sizes
+		add_filter( 'render_block_core/image', array( $this, 'fix_img_block_sizes' ), 10, 3 );
```

Comment:
```suggestion
		if ( ! is_admin() ) {
			add_filter( 'render_block_core/image', array( $this, 'fix_img_block_sizes' ), 10, 3 );
		}
```

and remove `is_admin()` check in `fix_img_block_sizes()`.